src/ProtocolV3TestBase.sol

```diff
@@ -307,6 +318,26 @@ contract ProtocolV3TestBase is CommonTestBase {
       / 100_00;
   }
 
+  function _isAboveBorrowCap(
+    IPool pool,
+    ReserveConfig memory borrowConfig,
+    uint256 borrowAmount
+  ) internal view returns (bool) {
+    if(keccak256(abi.encodePacked(borrowConfig.symbol)) == keccak256(abi.encodePacked('DAI')))
+      return false;
+
+    DataTypes.ReserveData memory reserveData = pool.getReserveData(borrowConfig.underlying);
+
+    uint256 scaledBorrowCap = borrowConfig.borrowCap * 10 ** borrowConfig.decimals;
+
+    uint256 currScaledVariableDebt = IVariableDebtToken(borrowConfig.variableDebtToken).scaledTotalSupply();
+    (,uint256 currTotalStableDebt,,) = IStableDebtToken(borrowConfig.stableDebtToken).getSupplyData();
+
+    uint256 totalDebt = currTotalStableDebt + currScaledVariableDebt.rayMul(reserveData.variableBorrowIndex);
+
+    return borrowAmount > (scaledBorrowCap - totalDebt);
```

Comment:
Usually safer to move the subtraction to an addition on the other side of inequalities to eliminate underflow reverts.